### PR TITLE
Add e2e tests for the Helm Chart

### PR DESCRIPTION
**Pull request by tomaszsek**
# Changes

- add Helm Chart lint test
- add Helm Chart e2e test

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes tests (if functionality changed/added)
- [ ] Includes docs (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md#commit-messages)

_See [the contribution guide](https://github.com/jenkinsci/kubernetes-operator/blob/master/CONTRIBUTING.md) for more details._

### CODE REVIEW

File: test/e2e/helm_test.go [hunk 1]
```diff
@@ -0,0 +1,60 @@
+// +build Helm
```
Comment (waveywaves) [inline test/e2e/helm_test.go:1]: @jitendar-singh Can you PTAL ?

File: config.base.env [hunk 2]
```diff
@@ -9,4 +9,5 @@ NAMESPACE=default
 API_VERSION=v1alpha2
 ALL_IN_ONE_DEPLOY_FILE_PREFIX=all-in-one
 GEN_CRD_API=gen-crd-api-reference-docs
-IMAGE_PULL_MODE=local
\ No newline at end of file
+IMAGE_PULL_MODE=local
+HELM_VERSION=3.1.2
```
Comment (waveywaves) [inline config.base.env:13]: Please add a new line to the EOF

**Comment by tomaszsek** [comment]
@waveywaves Do you have an example of using https://github.com/helm/chart-testing? Can we think about it in another PR?

File: config.base.env (hunk 2 above)
Comment (tomaszsek) [reply config.base.env:13]: Why do we need a new line here?

File: test/e2e/helm_test.go (hunk 1 above)
Comment (jitendar-singh) [reply test/e2e/helm_test.go:1]: Looks good

**Comment by tomaszsek** [comment]
@jitendar-singh I'm not sure if `$GOPATH/bin` is on the PATH in Travis.

**Comment by jitendar-singh** [comment]
I did test it @tomaszsek and the build was successful :)

**Pull request merged**